weepickle/src/main/scala/com/rallyhealth/weepickle/v1/LowPriorityImplicits.scala

```diff
@@ -17,19 +19,27 @@ abstract class LowPriorityImplicits
 
   private def toEnumerationName[E <: scala.Enumeration](e: E): To[E#Value] = {
     val cache = new ConcurrentHashMap[String, E#Value] // mitigate withName() slowness.
+    var lastVset: E#ValueSet = null
```

Comment:
Q: I thought I remember @jducoeur saying something about the default, is this form right or should it be this? 
```suggestion
    var lastVset: E#ValueSet = _
```

Reply:
Pretty sure it'd fail to compile. IIRC, only works for class fields. Also, it's getting phased out in scala 3 anyway.

Reply:
> Also, it's getting phased out in scala 3 anyway.

That was the part I was thinking, I just couldn't remember if it was being enforced or dropped in Scala 3.

Reply:
Yeah, the underscore initializer is sort of a horrible wart in the language.  In Scala 3 it becomes `uninitialized`.  In practice, it will wind up as `null` regardless, so calling a spade a spade isn't a terrible thing.

Reply:
> In Scala 3 it becomes `uninitialized`

_That's_ what I couldn't remember! Thank you!